build/NuSpecs/ProjectReunion-Nuget-Native.targets

```diff
@@ -9,7 +9,8 @@
     <Reference Include="$(MSBuildThisFileDirectory)..\..\lib\uap10.0\Microsoft.ProjectReunion.winmd">
       <Implementation>Microsoft.ProjectReunion.dll</Implementation>
     </Reference>
-    <ReferenceCopyLocalPaths Include="$(MSBuildThisFileDirectory)..\..\runtimes\win10-$(Native-Platform)\native\Microsoft.ProjectReunion.dll" />
-    <ReferenceCopyLocalPaths Include="$(MSBuildThisFileDirectory)..\..\runtimes\win10-$(Native-Platform)\native\Microsoft.ProjectReunion.pri" />
+    <!--  Figure out the right long term approach for VS to not pick these up when using them with a framework package. -->
+    <!-- <ReferenceCopyLocalPaths Include="$(MSBuildThisFileDirectory)..\..\runtimes\win10-$(Native-Platform)\native\Microsoft.ProjectReunion.dll" /> -->
```

Comment:
Isn't deletion the long-term solution?  These shouldn't be in the targets file?

Reply:
This comment is more a note to myself/other people. In the future if we want to ingest packages that also are meant to be consumed not through a framework package then they will have these types of lines in their targets files. We will want a programmatic solution, whether that is having a script that edits the targets to remove these types of things or some VS props/targets file magic.